pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -57,15 +59,17 @@ JoinMailingListPayload:
       description: The umbrella id of the mailing list the user is interested in (one of umbrellaId OR studyGuid but not both is required)
       example: cmi
       type: string
+      format: guid
     studyGuid:
       description: The studyGuid of the mailing list the user is interested in (one of umbrellaId OR studyGuid but not both is required)
       example: ANGIO
       type: string
+      format: guid
```

Comment:
Is this `guid` format something that is defined somewhere (in this doc or is a well-known thing)?

Reply:
Nope, it's not- the list of supported formats are defined [here](https://swagger.io/specification/#data-type-format), but if the value of the `format` property isn't one of the known values, tools are support to treat it as a open-string valued property that can have any value. From my understanding of the platform, the definition of `GUID` is pulling double-duty- as a set of types (Study GUID, Answer GUID, etc), and also as a general way of identifying a unique identifier type. This is using the latter definition, and my intent here is to more visibly call out the identifier types in the API outside of just their name.